src/components/display/DisplayImage.js

```diff
@@ -2,42 +2,37 @@ import axiosInstance from "../api/axios";
 import { useState, useEffect } from "react";
 import "./DisplayImage.css";
 
+let perPage = 0;
+
 const DisplayImage = ({ images, setImages }) => {
- 
-  // const [showMore, setShowMore] = useState(5);
+
+  const showImage = async () => {
+    try {
+      const res = await axiosInstance.get(`?&offset=${perPage}`);
+      setImages(res.data.results);
+    } catch (error) {
+      console.log(error);
+    }
+    perPage += 5;
```

Comment:
I think you should use the length of `res.data.results` to update this

Reply:
I dont get this length part? am I adding length method to get the response from the API? also what's the effect of this?

Reply:
Imagine if there are only two pages, and there are 5 results per page.

If at any time there are only 6 results in the dB, that means your backend would only return 1 result for the second page, using hard coded `perPage += 5` doesn't account for a situation, using the length of the array you're getting from the backend does.

Reply:
oh ok. I will adjust this.